integreat_cms/cms/forms/contacts/contact_form.py

```diff
@@ -38,12 +47,24 @@ class Meta:
             "phone_number",
             "mobile_phone_number",
             "website",
+            "opening_hours",
         ]
 
         error_messages = {
             "location": {"invalid_choice": _("Location cannot be empty.")},
         }
 
+    def __init__(self, **kwargs: Any) -> None:
+        adopt_hours = True
+        if "instance" in kwargs and kwargs["instance"] and kwargs["instance"].id:
+            adopt_hours = kwargs["instance"].opening_hours is None
+            if adopt_hours:
+                kwargs["instance"].opening_hours = kwargs[
+                    "instance"
+                ].location.opening_hours
```

Comment:
I find this a little hard to read, what do you think?
```suggestion
        instance = kwargs.get("instance")
        if instance and instance.id:
            adopt_hours = instance.opening_hours is None
            if adopt_hours:
                instance.opening_hours = instance.location.opening_hours
```

Reply:
Looks great and should be applied now :)